### ISSUE TITLE
Change `blitz generate pages` to use actual model attributes

### ISSUE THREAD

**Issue by flybayer** [blitz-js/blitz#259]
### What do you want and why?

Currently the `blitz generate` command assumes the model has a `name` attribute. But we need to change this to use the actual attributes on the generated model.

When generating pages without passing model attributes, we should show an inquirer multi-select prompt with the current model attributes.
```
blitz generate pages task
```

When generating pages with passed model attributes, we use those instead showing a prompt
```
blitz generate pages task name email
```

When generating everything with `all`, the generated pages will include the attributes passed:
```
blitz generate all task email:string:unique name:string? age:int
```

Related: https://github.com/blitz-js/blitz/issues/234 probably want to wait on this issues until that is completed.

**Comment by flybayer** [comment]
Closing in favor of https://github.com/blitz-js/blitz/issues/2038
